packages/strapi-admin/admin/src/components/LeftMenu/LeftMenuFooter/index.js

```diff
@@ -17,22 +16,22 @@ function LeftMenuFooter({ version }) {
   return (
     <Wrapper>
       <div className="poweredBy">
-        <a key="website" href="https://strapi.io" target="_blank" rel="noopener noreferrer">
+        <A key="website" href="https://strapi.io" target="_blank" rel="noopener noreferrer">
```

Comment:
Why not naming this Tag as Link?

Reply:
Because we are already using a component called `Link` from `react-router-dom` so naming it `A` helps differentiating it from the other one.